recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set name = "Flask-SQLAlchemy" %}
-{% set version = "3.0.3" %}
+{% set name = "flask_sqlalchemy" %}
```

Comment:
Please do not change the name

Reply:
The PyPI name (or at least download URL) seems to have changed - build was failing before I did this. I guess I should hardcode the `flask_sqlalchemy` name in `source.url` but leave it as `Flask-SQLAlchemy` here?